src/test/test-create-user.ts

```diff
@@ -0,0 +1,142 @@
+import axios from 'axios';
+import { expect } from 'chai';
+import { createdUser, expectResponse } from './input';
+import { createToken } from '../create-token';
+import { findUser } from '../find-user';
+
+describe('Testing createUser', () => {
+  it('should create an user and return it', async () => {
+    const url = 'http://localhost:4000';
+    const token = createToken(0, true);
+    const input = {
+      name: 'eu',
+      email: 'eu@gmail.com',
+      birthDate: '27/12/1900',
+      password: 'mudar123',
+    };
+    const response = await axios.post(
+      url,
+      {
+        query: createdUser,
+        variables: {
+          user: input,
+        },
+      },
+      {
+        headers: {
+          authorization: token,
+        },
+      },
+    );
+    const user = await findUser('eu@gmail.com');
```

Comment:
```suggestion
    const user = await findUser(input.email);
```

Lembrar da questão de evitarmos ter 2 verdades ou o mesmo valor definido em 2 lugares diferentes. Assim fica mais fácil de trocar e diminui o risco de gerar bugs.